README.md

```diff
@@ -60,6 +60,13 @@ the work provided in those for our format handling.
   - original author: A Grund @Flamefire
   - status: implemented (base standard for X-ray laser & density profiles)
 
+- UPIC-Emma (UCLA, USA)
+  - domain: 2-1/2D electromagnetic particle-in-cell code
+  - [repository](https://github.com/UCLA-Plasma-Simulation-Group/upic-emma-2.0)
```

Comment:
@RemiLehe can you please add the license in brackets as well?
Do we know the (abbreviated) name of the license in [UPIC-Emma](https://github.com/UCLA-Plasma-Simulation-Group/upic-emma-2.0/blob/master/license)?

 Just for curiosity, is this an [OSI-approved](https://opensource.org/licenses) license? Looks like a mix of a new-BSD/MIT-style license and a [CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) restriction.